File: 2011. Final Value of Variable After Performing Operations.js
/**
 * @param {string[]} operations
 * @return {number}
 */
var finalValueAfterOperations = function(operations) {
    count = 0;
    for (let i = 0; i < operations.length; i++){
        if (operations[i][0] == '-' || operations[i][2] == '-') count--;
        else count += 1;
    }
    return count;
};

File: 2011. Final Value of Variable After Performing Operations.py
class Solution:
    def finalValueAfterOperations(self, operations: List[str]) -> int:
        count = 0
        for o in operations:
            if (o[0] == '-' or o[2] == '-'):
                count -= 1
            elif (o[0] == '+' or o[2] == '+'):
                count += 1
        return count